utils.py

```diff
@@ -453,3 +454,20 @@ def get_all_language_codes_and_names():
 def unescape_encoded_uri_component(escaped_string):
     """Unescape a string that is encoded with encodeURIComponent."""
     return urllib.unquote(escaped_string).decode('utf-8')
+
+
+ASSET_DIR_PREFIX = None
+def get_asset_dir_prefix():
+    """Returns prefix for asset directory depending whether dev or prod.
+    It is used as a prefix in urls for images, css and script files.
+    """
+    global ASSET_DIR_PREFIX # pylint: disable=global-statement
+    if ASSET_DIR_PREFIX is None:
```

Comment:
Do "if not ASSET_DIR_PREFIX". I know this will recompute if ASSET_DIR_PREFIX = '' but I think that's OK; it will only do so for dev anyway.

Otherwise (and I've run into this already) we sometimes get ASSET_DIR_PREFIX set to '' and this doesn't change even after the minification env var is set to True (making feconf.IS_MINIFIED true).